spec/models/calculation_spec.rb

```diff
@@ -48,5 +48,13 @@
       let(:parameters) { {P1: 2} }
       it { is_expected.to eq 4 }
     end
+
+    context 'when pass `since` value and `from`, `to`, `unit` parameters' do
+      let(:value) { 'since' }
```

Comment:
Please check the function description in the ticket

Reply:
Thank you. I changed description.